restcomm/restcomm.http/src/main/java/org/restcomm/connect/http/CallsEndpoint.java

```diff
@@ -363,7 +363,8 @@ protected Response putCall(final String accountSid, final MultivaluedMap<String,
         final String to = data.getFirst("To").trim();
         final String username = data.getFirst("Username");
         final String password = data.getFirst("Password");
-        final Integer timeout = getTimeout(data);
+        Integer timeout = getTimeout(data);
+        timeout = timeout != null ? timeout : 30;
```

Comment:
i really would love to see this default timeout defined as a constant maybe in VI, or coming from conf...

Reply:
again in this PR, its just being set as per older implementation, i just set it here instead of multiple conditional statements bellow